### PR TITLE
Bump version to 20.10.0-beta1

### PR DESCRIPTION
**Pull request by tiborvass**
Signed-off-by: Tibor Vass <tibor@docker.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1 +1,164 @@
...
+
+### Swarm
+
++ Add support for swarm jobs [moby/moby#40307](https://github.com/moby/moby/pull/40307)
++ Add support for sending down service Running and Desired task counts [moby/moby#39231](https://github.com/moby/moby/pull/39231)
++ service: support --mount type=bind,bind-nonrecursive [moby/moby#38788](https://github.com/moby/moby/pull/38788)
++ Support ulimits on Swarm services. [moby/moby#41284](https://github.com/moby/moby/pull/41284) [docker/cli#2712](https://github.com/docker/cli/pull/2712)
++ Add support for capabilities options in services [moby/moby#39173](https://github.com/moby/moby/pull/39173)
++ Service cap-add/cap-drop: add special "RESET" value [docker/cli#2709](https://github.com/docker/cli/pull/2709)
+* Replace swarm Capabilites API with cap add/drop API [moby/moby#41249](https://github.com/moby/moby/pull/41249)
```
Comment (cpuguy83) [inline CHANGELOG.md:163]: I don't think this should be in the changelog, since the capabilities API was never released.

Comment (cpuguy83) [reply CHANGELOG.md:163]: Really all these capabilities things can be merged into 1.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1 +1,164 @@
...
++ Add dockerd-rootless-setuptool.sh [moby/moby#40950](https://github.com/moby/moby/pull/40950)
+* Support `--exec-opt native.cgroupdriver=systemd` [moby/moby#40486](https://github.com/moby/moby/pull/40486)
+
+### Security
+
+* Fix CVE-2019-14271 loading of nsswitch based config inside chroot under Glibc [moby/moby#39612](https://github.com/moby/moby/pull/39612)
+* seccomp: Whitelist `clock_adjtime`. `CAP_SYS_TIME` is still required for time adjustment [moby/moby#40929](https://github.com/moby/moby/pull/40929)
+* seccomp: Add openat2 and faccessat2 to default seccomp profile [moby/moby#41353](https://github.com/moby/moby/pull/41353)
+* seccomp: allow 'rseq' syscall in default seccomp profile [moby/moby#41158](https://github.com/moby/moby/pull/41158)
+* seccomp: Remove dependency in dockerd on libseccomp [moby/moby#41395](https://github.com/moby/moby/pull/41395)
```
Comment (cpuguy83) [inline CHANGELOG.md:149]: Move to packaging?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1 +1,164 @@
...
+
+### Runtime
+
++ Support cgroup2 [moby/moby#40174](https://github.com/moby/moby/pull/40174) [moby/moby#40657](https://github.com/moby/moby/pull/40657) [moby/moby#40662](https://github.com/moby/moby/pull/40662)
+* cgroup2: use "systemd" cgroup driver by default when available [moby/moby#40846](https://github.com/moby/moby/pull/40846)
++ new storage driver: fuse-overlayfs [moby/moby#40483](https://github.com/moby/moby/pull/40483)
++ Update containerd binary to v1.4.0 [moby/moby#40982](https://github.com/moby/moby/pull/40982)
++ `docker push` now defaults to `latest` tag instead of all tags [moby/moby#40302](https://github.com/moby/moby/pull/40302)
++ Added ability to change the number of reconnect attempts during connection loss while pulling an image by adding max-download-attempts to the config file [moby/moby#39949](https://github.com/moby/moby/pull/39949)
++ Add support new built-in runtime for using the containerd v2 shim by using the runtime `com.docker.runtime.runc.v2` [moby/moby#41182](https://github.com/moby/moby/pull/41182)
```
Comment (AkihiroSuda) [inline CHANGELOG.md:99]: com.docker.runtime.runc.v2 -> io.containerd.runc.v2

Comment (AkihiroSuda) [reply CHANGELOG.md:99]: Also better to clarify v2 is the default now

File: CHANGELOG.md [hunk 4]
```diff
@@ -1 +1,164 @@
...
+* Add "context" to "docker version" and "docker info" [docker/cli#2500](https://github.com/docker/cli/pull/2500)
+* Propagate platform flag to container create API [docker/cli#2551](https://github.com/docker/cli/pull/2551)
+* The `docker ps --format` flag now has a `.State` placeholder to print the container's state without additional details about uptime and health check [docker/cli#2000](https://github.com/docker/cli/pull/2000)
+* Add support for docker-compose schema v3.9 [docker/cli#2073](https://github.com/docker/cli/pull/2073)
+* Add support for docker push --quiet [docker/cli#2197](https://github.com/docker/cli/pull/2197)
+* Hide flags that are not supported by BuildKit, if BuildKit is enabled [docker/cli#2123](https://github.com/docker/cli/pull/2123)
+* Update flag description for `docker rm -v` to clarify the option only removes anonymous (unnamed) volumes [docker/cli#2289](https://github.com/docker/cli/pull/2289)
+* Improve tasks printing for docker services [docker/cli#2341](https://github.com/docker/cli/pull/2341)
+* Use ssh multiplexing [docker/cli#2132](https://github.com/docker/cli/pull/2132)
```
Comment (tianon) [inline CHANGELOG.md:72]: I think this one was reverted: https://github.com/docker/cli/pull/2303 :sweat_smile:

**Review by cpuguy83 — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 5]
```diff
@@ -1 +1,160 @@
 # Changelog
+
+For official release notes for Docker Engine CE, visit the
+[release notes page](https://docs.docker.com/engine/release-notes/).
+
+## 20.09.0
```
Comment (thaJeztah) [inline CHANGELOG.md:6]:
```suggestion
## 20.10.0
```

**Review by thaJeztah — APPROVED** [review]
LGTM

**Comment by AkihiroSuda** [comment]
Could you create git tags?

**Comment by khs1994** [comment]
next version is 20.10?

File: CHANGELOG.md [hunk 6]
```diff
@@ -1 +1,160 @@
...
+* Deprecate host-discovery and overlay networks with external k/v stores [moby/moby#40614](https://github.com/moby/moby/pull/40614) [moby/moby#40510](https://github.com/moby/moby/pull/40510)
+* Deprecate Dockerfile legacy 'ENV name value' syntax, use `ENV name=value` instead [docker/cli#2743](https://github.com/docker/cli/pull/2743)
+* Remove deprecated "filter" parameter for API v1.41 and up [moby/moby#40491](https://github.com/moby/moby/pull/40491)
+* Disable distribution manifest v2 schema 1 on push [moby/moby#41295](https://github.com/moby/moby/pull/41295)
+* Remove hack MalformedHostHeaderOverride breaking old docker clients (<= 1.12) in which case, set `DOCKER_API_VERSION` [moby/moby#39076](https://github.com/moby/moby/pull/39076)
+* Remove "docker engine" subcommands [docker/cli#2207](https://github.com/docker/cli/pull/2207)
+* Remove experimental "deploy" from "dab" files [docker/cli#2216](https://github.com/docker/cli/pull/2216)
+* Remove deprecated `docker search --automated` and `--stars` flags [docker/cli#2338](https://github.com/docker/cli/pull/2338)
+* No longer allow reserved namespaces in engine labels [docker/cli#2326](https://github.com/docker/cli/pull/2326)
```
Comment (cpuguy83) [inline CHANGELOG.md:21]: I think this list is missing the removal of `--stream`, possibly.

Comment (thaJeztah) [reply CHANGELOG.md:21]: Yes we should add (let me also open a PR for the `deprecated.md`. Related pull requests are https://github.com/docker/cli/pull/2105 and https://github.com/moby/moby/pull/39983

Comment (thaJeztah) [reply CHANGELOG.md:21]: opened https://github.com/docker/cli/pull/2809

**Pull request merged**